### PR TITLE
Add snap version to details page

### LINKED ISSUE #14010: Display Snap version Snap details page
We should display the version on the Snap details page. Currently it's only visible on the Snap cards on the main Snaps settings page.

### PR DESCRIPTION
**Pull request by FrederikBolding**
## Explanation

Added snap version to snap details page

## More information

Fixes #14010 

### Before

image

### After

image

## Manual testing steps

- Go to https://filsnap.chainsafe.io/
- Click "Connect to MetaMask"
- Accept the prompts to install the snap
- Open MM
- Go to snap settings
- Click "see details"
- Verify that you can see installed version etc

### CODE REVIEW

File: ui/pages/settings/flask/view-snap/view-snap.js [hunk 1]
```diff
@@ -71,6 +72,10 @@ function ViewSnap() {
   if (!snap) {
     return null;
   }
+
+  const versionHistory = snap.versionHistory ?? [];
+  const firstInstall = versionHistory[0];
```
Comment (darkwing) [inline ui/pages/settings/flask/view-snap/view-snap.js:77]: I've been trying to get the codebase in the habit of:

```
const [firstInstall] = versionHistory;
```

File: ui/pages/settings/flask/view-snap/index.scss [hunk 2]
```diff
@@ -24,6 +24,19 @@
     }
   }
 
+  &__install-details {
+    border-bottom: 1px solid var(--color-border-muted);
+    margin-right: 24px;
```
Comment (darkwing) [inline ui/pages/settings/flask/view-snap/index.scss:29]: We might want to try `margin-inline-end` for RTL support.

**Review by darkwing — DISMISSED** [review]

File: ui/pages/settings/flask/view-snap/view-snap.js (hunk 1 above)
Comment (FrederikBolding) [reply ui/pages/settings/flask/view-snap/view-snap.js]: Done 👍

File: ui/pages/settings/flask/view-snap/index.scss (hunk 2 above)
Comment (FrederikBolding) [reply ui/pages/settings/flask/view-snap/index.scss]: Done 👍

**Review by mcmire — APPROVED** [review]
Looks good!

**Review by ryanml — APPROVED** [review]
lgtm!

**Pull request merged**